src/jit/codegenxarch.cpp

```diff
@@ -5915,6 +5918,15 @@ void CodeGen::genCallInstruction(GenTreePtr node)
             break;
         }
     }
+
+    // Is the caller supposed to pop the arguments?
+    if ((call->gtFlags & GTF_CALL_POP_ARGS) != 0)
```

Comment:
Nit: could be simply `if ((call->gtFlags & GTF_CALL_POP_ARGS) != 0 && stackArgBytes != 0)`